agreement/voteTracker.go

```diff
@@ -116,8 +117,21 @@ func (tracker *voteTracker) handle(r routerHandle, p player, e0 event) event {
 		}
 
 		sender := e.Vote.R.Sender
-		_, equivocator := tracker.Equivocators[sender]
+		eqVote, equivocator := tracker.Equivocators[sender]
 		if equivocator {
+			equivocationDetails := telemetryspec.EquivocatedVoteEventDetails{
```

Comment:
this seems redundant, what extra information does logging the second, third equivocations give us?

Reply:
It gives us -
1. the proposal value. ( which might not be identical to any of the previous ones )
2. the ability to count the number of equivocated votes we received

This doesn't happen in normal operations, so I don't really care about performance. But we want to extract as much information as we can for further analysis.

Reply:
A funny notable behavior I just found, when browsing the code:
- assuming we always try filtering before sending a voteAccepted event, any 2nd+ equivocation should be filtered, and won't enter this logic in the first place. (we should probably write a test to cover this case)
- so if we ever see this telemetry event, something is very odd.

Reply:
You're correct. I also need to add this event in case of the filtering.. since the goal here is to identify the use cases where we have equivocated votes.

Reply:
@tsachiherman While it shouldn't happen in normal operations, a misconfigured node can cause this to happen very often.  Mostly this should identify if someone has accidentally put the partkey on multiple nodes -- or worse stolen it and is using it maliciously.  I do think the expectations of > 1 equivocations goes down exponentially.

Reply:
> I do think the expectations of > 1 equivocations goes down exponentially.

That's not true--you can repeatedly equivocate with the same set of credentials and create as many as you wish.